content/docs/start/ml-pipelines.md

```diff
@@ -0,0 +1,247 @@
+---
+title: 'Get Started: ML Pipelines'
+---
+
+# Get Started: ML Pipelines
+
+DVC makes it easy to iterate on your ML project using Git commits, tags, or
+branches. You can try different ideas quickly by tuning
```

Comment:
`You can try different ideas quickly by tuning` feels like it appeals more to the experiments section now, message here should be a bit different to my mind